fileio/fileio.go

```diff
@@ -195,3 +195,21 @@ func ReadFileToSingleLineString(filename string) string {
 	}
 	return catInput
 }
+
+func WriteToFileHandle(file io.Writer, rec string) {
```

Comment:
I don't think this function is necessary. It would usually be easier and faster to just do `file.Write([]byte(rec))`

Reply:
I don't follow how I could call the function within itself as a helper function. Could you expand?